wiz-admission-controller/values.yaml

```diff
@@ -151,6 +151,52 @@ imageIntegrityWebhook:
 
   patchImageDigestAnnotation: true # If true, annotate the pod with mapping between tag->digest.
 
+# Kubernetes audit logs webhook is currently in private preview.
+# This webhooks sends the audit logs to Wiz, and should never block any requests.
+kubernetesAuditLogsWebhook:
+  enabled: false 
+
+    # Override to run admission controller on specific resources.
+  rules: # https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-rules
+    - operations:
+      - "*"
+      apiGroups:
+      - "*"
+      apiVersions:
+      - "*"
+      resources:
+        - "namespaces"
+        - "secrets"
+        - "configmaps"
+        - "storageclasses"
+        - "persistentvolumes"
+        - "persistentvolumeclaims"
+        - "controllerrevisions"
+        - "ingresses"
+        - "services"
+        - "networkpolicies"
+        - "pods"
+        - "deployments"
+        - "jobs"
+        - "cronjobs"
+        - "replicasets"
+        - "statefulsets"
+        - "daemonsets"
```

Comment:
There is this one deprecated controller called ReplicationController that can also serve as a workload, lets add it as well:
```
apiVersion: v1
kind: ReplicationController
metadata:
  name: nginx
spec:
  replicas: 3
  selector:
    app: nginx
  template:
    metadata:
      name: nginx
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx
        image: nginx
        ports:
        - containerPort: 80
```

Reply:
OK, added